include/ray/ray.h

```diff
@@ -32,6 +32,9 @@ class FnInfo {
   const std::vector<WorkerId>& workers() const {
     return workers_;
   }
+  void remove_worker(WorkerId workerid) {
```

Comment:
Nope, I had it originally for the case when we were removing the workers one-by-one, I just didn't bother deleting it. Should I?